lib/tasks_private/spec_helper.rake

```diff
@@ -445,6 +445,9 @@ class PopulateTower
     }
     nonexistent_project = create_obj(uri, data)
     @tower_data[:items][data[:name]] = { :id => nonexistent_project['id'] }
+
+    sleep(DEL_SLEEP)
```

Comment:
A request: can you make the value to this `DEL_SLEEP` and 2 other parameters (Line 61) to be overridable by env variable?
These values are dependent to the setup.

Reply:
@jameswnl do you mean `sleep(ENV["DEL_SLEEP"] || DEL_SLEEP)` ?

Reply:
let's do it in the declaration line: 
`DEL_SLEEP = ENV["DEL_SLEEP"] || 20`

Reply:
done